modules/pool/pool.prisma

```diff
@@ -65,7 +66,7 @@ model PrismaPoolLinearData {
 
     id                  String
     poolId              String
-    pool                PrismaPool          @relation(fields:[poolId, chain], references: [id, chain])
+    pool                PrismaPool          @relation(fields:[poolId, chain], references: [id, chain], onDelete: Cascade)
```

Comment:
Since there are delete cascade now, maybe we can also simplify `poolService.deletePool()`?